setup.py

```diff
@@ -118,32 +52,18 @@ def load_long_description():
         description=pyproject["project"]["description"],
         long_description=load_long_description(),
         long_description_content_type='text/markdown',
-        cmdclass=cmdclass,
         zip_safe=False,
-        setup_requires=pyproject["build-system"]["requires"],
-        install_requires=pyproject["project"]["dependencies"],
         extras_require=pyproject["project"]["optional-dependencies"],
-        package_data={
-            'deeptime.data': ['data/*.npz'],
-            'deeptime.src.include': ['*.h'],
-        },
+        packages=find_packages(where="."),
+        package_dir={"deeptime": "deeptime", "versioneer": "."},
+        cmake_install_dir="deeptime/",
+        cmake_args=cmake_args,
         include_package_data=True,
         python_requires=pyproject["project"]["requires-python"],
+        ext_modules=[eig_qr_extension()]
     )
 
-
-def configuration(parent_package='', top_path=None):
-    from numpy.distutils.misc_util import Configuration
-    config = Configuration(None, parent_package, top_path)
-    config.set_options(ignore_setup_xxx_py=True,
-                       assume_default_configuration=True,
-                       delegate_options_to_subpackages=True)
-    config.add_subpackage('deeptime')
-    return config
-
-
 if __name__ == '__main__':
-    from numpy.distutils.core import setup
-
-    metadata['configuration'] = configuration
+    for i in (Path(__file__).resolve().parent / "_skbuild").rglob("CMakeCache.txt"):
```

Comment:
This stuff is worth a comment for future generations :)
What is being replaced here and why.

Reply:
I'll put a comment there. For reference it is from scikit-build/scikit-build#521; if you build the library locally there is an `_skbuild` subdirectory in which the cmake cache sits and if you subsequently try to build the library via conda you might get a cache mismatch. But I guess it should be removed as soon as the issue is fixed and a new skbuild release is out.

Reply:
So it is already an optimization workaround?

Reply:
it's a workaround you only need if you first build the pip package and subsequently a conda package without removing the local build cache first :slightly_smiling_face:

Reply:
I mean in principle I think a local cache directory should have absolutely no influence on a subsequent conda build but unfortunately it does :confused: